docs/ifaq.md

```diff
@@ -0,0 +1,13 @@
+#IFAQ - Infrequently Asked Questions
```

Comment:
This doesn't render correctly:

https://github.com/tirodkar/EarlGrey/blob/7e5b64c693082b26628d481a1b8e39232474b0f3/docs/ifaq.md

Also are you planning on linking this file to README or from somewhere in FAQ?